deployment/united-manufacturing-hub/values.yaml

```diff
@@ -2185,6 +2197,38 @@ kafkatoblob:
     enabled: true
     minAvailable: 1
 
+
+### kafka-topic-updater ###
+
+kafkatopicupdater:
+  image:
+    repository: unitedmanufacturinghub/kafka-topic-updater
+    pullPolicy: IfNotPresent
+  env:
+    # area and productionline are variables inserted into the topic structure to fill up the old structure to the new one
+    area: area
```

Comment:
Change these to DefaultArea, DefaultProductionLine to match our placeholders in factoryinsight

Reply:
![image](https://user-images.githubusercontent.com/6317420/197496921-70a15e97-07d7-4e1e-82ad-c01364cb56d1.png)